### ISSUE TITLE
Display "Terms of Service" agreement in other formats

### ISSUE THREAD

**Issue by kimcoleman** [strangerstudios/paid-memberships-pro#870]
Currently we display the "Terms of Service" area as a textarea/frame that loads the content of the assigned "Terms of Service" page (under Settings > Advanced Settings). 

A user has requested to hide the textarea/frame and instead simply show a checkbox with link to view the TOS page in a new window. For example:

[ ] I agree to the {LINKED TO TOS PAGE}.

**Comment by martuitam** [comment]
Yes, please. And for Europe, or at least some countries in Europe, in order to be GDPR compliance we need to include a short text offering basic information on who is the responsible for the data, the reason to keep it, the rights and so on. 

What I am doing actually is putting that essential info on a page that I'm showing there instead of my actual TOS.

Then, I have translated the checkbox text including links to my actual TOS and Privacy Policy.

**Comment by grifini** [comment]
Any improvement on this topic? Currently paid membership pro provides only one terms agreements. But most sites want to add privacy and terms confirmations both. Is there any way to do it with paidmembership pro?

**Comment by dparker1005** [comment]
Thanks for opening this. Closing this as addressed in PMPro 3.1.

The checkout Terms of Service field now displays as a checkbox with a link to the assigned TOS page, instead of embedding the full page content on the checkout page. The checkbox label can also be customized with the `pmpro_tos_field_label` filter, and additional checkout fields can be added after the TOS area if a site needs separate privacy or consent fields.
